setup-scripts/health-checker.py

```diff
@@ -227,7 +226,7 @@ async def check_health():
         failed_records_file = json.dumps(failed_records, indent=2)
 
     # send a message if we force notification, there is a failures dump or just once daily (heartbeat) on 1 AM
-    if force_notify or failed_records_file or datetime.utcnow().hour == 1:
+    if force_notify or json_check["disabled"] or failed_records_file or datetime.utcnow().hour == 1:
```

Comment:
I don't get it. I thought we wanted to **not** send a message if the portal is manually disabled.
This code makes it so `json_check["disabled"]` is treated essentially the same as `force_notify`
and thus will notify us still. Right?

Reply:
nope, it will send the message but not add a @skynet-prod ping which is something we don't need every hour

Reply:
Oh I get how `force_notify` works now. Tanks! I thought it had to do with the actual sending of the message, not the notification